.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "libjxl-src/submodules/libjxl"]
 	path = libjxl-src/submodules/libjxl
 	url = https://github.com/libjxl/libjxl.git
+	branch = v0.5.x
```

Comment:
Wait, there is no actual submodule update. This line is not really helpful, please do this:

```
pushd submodules/libjxl
git remote update && git checkout v0.5.x
popd
git commit -a -m "Update libjxl to v0.5.x"
```

Reply:
Right, it looked too easy. Now tests [failed](https://github.com/stan-kondrat/jxl-rs/runs/3277644528?check_suite_focus=true). Unfortunately I can't investigate issues, on my local machine build doesn't work at all.

It would be nice if you could update libjxl